mmv1/products/networksecurity/MirroringDeployment.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'MirroringDeployment'
+description: MirroringDeployment represents the collectors within a Zone and is associated with a deployment group.
+min_version: 'beta'
+docs:
```

Comment:
Could you add docs links for these resources?

Reply:
Done.

Reply:
Actually docs doesn't seem to support a native url field:
https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/api/resource/docs.go
And I can't find any other resource using docs for these links, so added it under 'references' instead following the convention from other resources.